android/app/src/main/java/app/organicmaps/search/TabAdapter.java

```diff
@@ -11,6 +12,8 @@
 import androidx.viewpager.widget.ViewPager;
 
 import com.google.android.material.tabs.TabLayout;
+
+import app.organicmaps.MwmApplication;
```

Comment:
Is it strictly necessary?

Reply:
Yes @biodranik, to access the static method `prefs`